spec/models/chat_channel_spec.rb

```diff
@@ -16,7 +16,14 @@
   describe "#allowed_user_ids" do
     it "is correct for each channel type" do
```

Comment:
I feel like we need more tests with better names here. Basically what we to test:
- direct message has only the allowed user ids of the users in the channel
- a public channel has not allowed user ids
- a private category channel has only staff members as allowed user ids